Document FIPS mode configuration for App Service Environment

sureddy1 opened the pull request:
Added instructions for enabling FIPS mode in App Service Environment, including JSON configuration. This applies to linux workloads only running in App Service Environment.

sureddy1 committed:
Document FIPS mode configuration for App Service Environment

Added instructions for enabling FIPS mode in App Service Environment, including JSON configuration.

stevenernst131 commented on articles/app-service/environment/app-service-app-service-environment-custom-settings.md:
```diff
@@ -103,6 +103,23 @@ To configure your App Service Environment to use just the ciphers that it requir
 > [!WARNING]
 > If incorrect values are set for the cipher suite that SChannel can't understand, all TLS communication to your server might stop functioning. In such a case, you'll need to remove the *FrontEndSSLCipherSuiteOrder* entry from **clusterSettings** and submit the updated Resource Manager template to revert back to the default cipher suite settings. Use this functionality with caution.
 
+## Enable FIPS mode
```
Can you make sure to emphasize earlier in this section that this specifically applies to Linux?

seligj95 replied:
+1 to this

sureddy1 replied:
Fixed

sureddy1 committed:
Specify FIPS mode applicability for Linux workloads

Clarified that FIPS mode applies specifically to Linux-based workloads in App Service Environment.
